iofog-agent-daemon/src/main/java/org/eclipse/iofog/network/IOFogNetworkInterface.java

```diff
@@ -172,4 +172,14 @@ private static void setDockerBridgeInterfaceName() throws Exception {
 
         return null;
     }
+    public static String getHostName() {
+        String hostname = "";
+        try {
+            InetAddress ip = InetAddress.getLocalHost();
+            hostname = ip.getHostName();
+        } catch (UnknownHostException e) {
+            LoggingService.logWarning(MODULE_NAME, "Unable to Get hostname : " +  e.getMessage());
```

Comment:
`Unable to Get hostname` ... should be `Unable to get hostname`

Reply:
Fixed